plugins/chat/spec/requests/core_ext/users_controller_spec.rb

```diff
@@ -52,5 +52,19 @@
       notifications = response.parsed_body["notifications"]
       expect(notifications.size).to eq(0)
     end
+
+    it "shows unread notifications even if the bookmark has been deleted if they have bookmarkable data" do
```

Comment:
Just curious, how is chat extending the controller such that we need to duplicate the test in the plugin as well?

Reply:
Yeah fair enough, I am not sure why I did this now...will remove